controller/RestSupportedVm.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoSync\controller;
+
+use oat\taoSync\model\VirtualMachine\SupportedVmService;
+use oat\taoTestCenter\controller\AbstractRestController;
+
+/**
+ * Class RestSupportedVm
+ * @package oat\taoSync\controller
+ *
+ * @OA\Info(title="TAO Supported list of VM versions API", version="0.1")
+ */
+class RestSupportedVm extends AbstractRestController
```

Comment:
Is there a reason to extend from `oat\taoTestCenter\controller\AbstractRestController`, rather than `\tao_actions_RestController`?